config/example/node.env

```diff
@@ -14,7 +14,7 @@ NEOFS_PROMETHEUS_SHUTDOWN_TIMEOUT=15s
 NEOFS_NODE_WALLET_PATH=./wallet.json
 NEOFS_NODE_WALLET_ADDRESS=NcpJzXcSDrh5CCizf4K9Ro6w4t59J5LKzz
 NEOFS_NODE_WALLET_PASSWORD=password
-NEOFS_NODE_ADDRESSES="s01.neofs.devenv:8080 /dns4/s02.neofs.devenv/tcp/8081 grpc://127.0.0.1:8082 grpcs://localhost:8083"
+NEOFS_NODE_ADDRESSES="s01.neofs.devenv:8080,/dns4/s02.neofs.devenv/tcp/8081,grpc://127.0.0.1:8082,grpcs://localhost:8083"
```

Comment:
Does it work with spaces now? Did it work previously?

Reply:
We should always think of compatibility. If this worked with spaces it should continue working with spaces.